packages/tailwind/src/utils/css/sanitize-declarations.ts

```diff
@@ -0,0 +1,30 @@
+import type { AtRule, Root, Rule } from "postcss";
+
+const regex = /rgb\(\s*(\d+)\s*(\d+)\s*(\d+)(?:\s*\/\s*([\d%.]+))?\s*\)/g;
+
+/**
+ * Meant to do all the things necessary, in a per-declaration basis, to have the best email client
+ * support possible.
+ *
+ * Currently it only converts all `rgb` declaration values that use a space-based syntax
+ * into a command based one since the space-based isn't well supported.
+ */
+export const sanitizeDeclarations = (
+  nodeContainingDeclarations: Rule | AtRule | Root,
+) => {
+  nodeContainingDeclarations.walkDecls((declaration) => {
+    declaration.value = declaration.value.replaceAll(
+      regex,
+      (_match, r, g, b, a) => {
+        const alpha = a === "1" || typeof a === "undefined" ? "" : `,${a}`;
+        return `rgb(${r},${g},${b}${alpha})`;
+      },
+    );
+
+    /*
+      We reset the regex's last index so that the state of the last time it ran doesn't intefere
+      with when it is used another time
+    */
+    regex.lastIndex = 0;
```

Comment:
can this be fixed by moving the regex declaration inside this function so we scope it to the function's lifetime?

Reply:
Definitely, but it might have a bit of a performance hit, since this function is called a bunch of times, might not be 100% worth the better readability